crates/moonbuild/src/entry.rs

```diff
@@ -321,101 +328,132 @@ pub fn run_test(
         }
     }
 
-    let mut passed = 0;
-    let mut failed = 0;
-    let mut runtime_error = false;
-    let mut expect_failed = false;
-    let mut apply_expect_failed = false;
+    let passed = Arc::new(AtomicU32::new(0));
+    let failed = Arc::new(AtomicU32::new(0));
+    let runtime_error = Arc::new(AtomicBool::new(false));
+    let expect_failed = Arc::new(AtomicBool::new(false));
+    let apply_expect_failed = Arc::new(AtomicBool::new(false));
+
+    let runtime = tokio::runtime::Builder::new_multi_thread()
+        .enable_all()
+        .build()?;
+
+    let mut handlers = vec![];
 
     for runnable_artifact in runnable_artifacts.iter() {
         let p = Path::new(runnable_artifact);
 
+        let passed = passed.clone();
```

Comment:
use `Arc::clone` instead of `.clone`?

Reply:
seems no difference here, ide inlay hints will tell us it's `Arc` wrapped